chamber/models/fields.py

```diff
@@ -234,22 +232,25 @@ class PriceField(DecimalField):
 
     def __init__(self, *args, **kwargs):
         self.currency = kwargs.pop('currency', ugettext('CZK'))
-        default_kwargs = {
-            'decimal_places': 2,
-            'max_digits': 10,
-            'humanized': lambda val, inst, field: price_humanized(val, inst, currency=field.currency)
-        }
-        default_kwargs.update(kwargs)
-        super().__init__(*args, **default_kwargs)
+        super().__init__(*args, **{
+            **kwargs,
+            **{
+                'decimal_places': 2,
+                'max_digits': 10,
+                'humanized': lambda val, inst, field: price_humanized(val, inst, currency=field.currency)
+            }
+        })
 
     def formfield(self, **kwargs):
-        default_kwargs = {
-            'form_class': chamber_fields.PriceField,
-            'currency': self.currency,
-        }
-        default_kwargs.update(kwargs)
+        return super(DecimalField, self).formfield(
```

Comment:
tady jsi vrátil python 2 super :trollface:

Reply:
díky, něco jsem kopíroval z KNJ a uteklo mi to